pkg/volume/csi/csi_plugin.go

```diff
@@ -211,6 +212,7 @@ func (p *csiPlugin) Init(host volume.VolumeHost) error {
 				if p.csiDriverLister == nil {
 					klog.Error(log("CSIDriverLister not found on KubeletVolumeHost"))
 				}
+				nodeLister = kletHost.CSINodeLister()
```

Comment:
this should be outside the csidriver feature gate check and under the csinode feature gate